### ISSUE TITLE
divj

### ISSUE THREAD

**Issue by bergerrjf** [qmcurrents/gimic#242]
Hello all!
I have tried "calc=divj" with various settings for which "calc=cdens" was perfectly working. I always get errors like: 

<<< ERROR: gimic(): Unknown operation! >>>
 * Deallocated grid data
 INFO: Deallocated basis set and atom data

Any help would be greatly appreaciated!
 Thanks and kind regards
  Raphael

**Comment by heikef** [comment]
This function has been switched off, it is not part of gimic any more. We can think about reactivating it. When we removed it nobody complained. It had been removed due to a larger clean up activity of the code.

**Comment by bergerrjf** [comment]
I think its useful, but in the end _you_ have to weigh pros and cons of resurrecting it.

**Comment by heikef** [comment]
Why is it useful for you?

**Comment by bergerrjf** [comment]
I am working on a theoretical description of gauges that lead to divergence-less current contributions.


________________________________

**Comment by heikef** [comment]
Ok, we put it on the to do list.

**Comment by bergerrjf** [comment]
great! thank you very much!!



_________________________________________________

   PD Dr. Raphael J. F. Berger, FRSC
   Materialchemie
   Paris-Lodron Universität Salzburg
   Jakob-Harringerstr. 2a
   A-5020 Salzburg
   ÖSTERREICH

   Room: 4OG/07
       Tel.: (+)43/ (0)662 8044 6260
____________________________________________________

________________________________

**Comment by heikef** [comment]
I assume you are after divJ doing a 3D type calculation?

**Comment by bergerrjf** [comment]
yes, all three components are important. it would be good with the read grid option. which iskond of mynew standard method after the base/std grid were disfunctioned.
________________________________

**Comment by heikef** [comment]
I understand.
